packages/gamepads_android/pubspec.yaml

```diff
@@ -0,0 +1,26 @@
+name: gamepads_android
+description: Android implementation of gamepads, a Flutter plugin to handle gamepad input across multiple platforms.
+version: 0.1.1
+homepage: https://github.com/flame-engine/gamepads
+repository: https://github.com/flame-engine/gamepads/tree/main/packages/gamepads_android
```

Comment:
If I remember correctly only one of these fields should be specified

Reply:
Consistent with other versions ATM 

https://github.com/flame-engine/gamepads/blob/main/packages/gamepads_ios/pubspec.yaml
https://github.com/flame-engine/gamepads/blob/main/packages/gamepads_darwin/pubspec.yaml
https://github.com/flame-engine/gamepads/blob/main/packages/gamepads_windows/pubspec.yaml
https://github.com/flame-engine/gamepads/blob/main/packages/gamepads_linux/pubspec.yaml